regression/cbmc/dynamic-object-01/test-no-cp.desc

```diff
@@ -0,0 +1,10 @@
+FUTURE
+main.c
+--no-simplify --no-propagation
+^EXIT=0$
+^SIGNAL=0$
+--
+^warning: ignoring
+--
+Check that the dynamic object property is havoc'd when dynamic objects are
```

Comment:
As usual - I'd suggest all tests with FUTURE or KNOWNBUG should have some kind of JIRA reference for when they are to be turned on, otherwise people will just forget about them

Reply:
In this case - shouldn't `EXIT=10`?

Reply:
Added JIRA ticket references. Also changed it to `EXIT=0`.